src/styles/card_style.scss

```diff
@@ -0,0 +1,92 @@
+@import 'variables_card-style';
+@import 'card-rating';
+
+.card {
+  width: $card-width;
+  height: 408px;
+  font-family: inherit;
+  padding: 16px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $color-white;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  transition: all 300ms linear;
```

Comment:
Applying transition to all properties can cause performance issues. It would be better to specify which properties you want to apply the transition to.